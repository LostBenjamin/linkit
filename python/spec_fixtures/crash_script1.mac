# variables are immutable
$var 100
$var 200
<var>

