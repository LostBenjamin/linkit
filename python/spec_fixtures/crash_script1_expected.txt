The resolved value 'var' is being changed to '200'

